src/tests/ftest/launch.py

```diff
@@ -684,13 +811,19 @@ def replace_yaml_file(yaml_file, args, yaml_dir):
         yaml_file (str): test yaml file
         args (argparse.Namespace): command line arguments for this program
         yaml_dir (str): directory in which to write the modified yaml files
+        vmd_flag (bool): PCI address includes VMD address (True)
+                         PCI address doesn't include VMD address (False).
+                         Defaults to False
 
     Returns:
         str: the test yaml file; None if the yaml file contains placeholders
             w/o replacements
+        env_vars (dict): Returns environment variable dictionary. Presently,
+            returns DAOS_DISABLE_VMD: "False"/"True" or empty dictionary.
 
     """
     replacements = {}
+    env_vars = {}
```

Comment:
To ensure we 'reset' the DAOS_DISABLE_VMD setting in each loop when a test does not use a bdev_list:
```suggestion
    env_vars = {"DAOS_DISABLE_VMD": "True"}
```

Reply:
done.